### PR TITLE
add embedding, bigbird api docstring

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

Others
### PR changes

Docs
### Describe

Add embedding, bigbird api docstring

### CODE REVIEW

File: paddlenlp/embeddings/token_embedding.py [hunk 1]
```diff
@@ -272,23 +292,29 @@ def _calc_word(self, word_a, word_b, calc_kernel):
     def dot(self, word_a, word_b):
         """
         Calculate the scalar product of 2 words.
```
Comment (ZeyuChen) [inline paddlenlp/embeddings/token_embedding.py]: `scalar product` -> `dot product`

File: paddlenlp/embeddings/token_embedding.py [hunk 2]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
     by specifying extended_vocab_path.
 
     Args:
-        embedding_name (object: `str`, optional, default to `w2v.baidu_encyclopedia.target.word-word.dim300`):
+        embedding_name (`str`, optional):
             The pre-trained embedding model name. Use `paddlenlp.embeddings.list_embedding_name()` to
-            show which embedding model we have alreaady provide.
-        unknown_token (object: `str`, optional, default to `[UNK]`):
+            show which embedding model we have alreaady provide.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: to show which embedding model we have alreaady provide. 
->
to list the names of all embedding models that we provide

File: paddlenlp/embeddings/token_embedding.py [hunk 3]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
...
+        embedding_name (`str`, optional):
             The pre-trained embedding model name. Use `paddlenlp.embeddings.list_embedding_name()` to
-            show which embedding model we have alreaady provide.
-        unknown_token (object: `str`, optional, default to `[UNK]`):
+            show which embedding model we have alreaady provide. 
+            Default to `w2v.baidu_encyclopedia.target.word-word.dim300`.
+        unknown_token (`str`, optional):
             Specifying unknown token as unknown_token.
-        unknown_token_vector (object: list, optional, default to `None`):
+            Default to `[UNK]`.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: Specifying unknown token as unknown_token.  这句话有点问题

File: paddlenlp/embeddings/token_embedding.py [hunk 4]
```diff
@@ -272,23 +297,29 @@ def _calc_word(self, word_a, word_b, calc_kernel):
...
         Calculate the cosine similarity of 2 words.
+
         Args:
-            word_a (object: `str`, required): The first word string.
-            word_b (object: `str`, required): The second word string.
+            word_a (`str`): The first word string.
+            word_b (`str`): The second word string.
+
         Returns:
             The cosine similarity of 2 words.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: specify the return type

File: paddlenlp/embeddings/token_embedding.py [hunk 5]
```diff
@@ -272,23 +297,29 @@ def _calc_word(self, word_a, word_b, calc_kernel):
...
         Calculate the scalar product of 2 words.
+
         Args:
-            word_a (object: `str`, required): The first word string.
-            word_b (object: `str`, required): The second word string.
+            word_a (`str`): The first word string.
+            word_b (`str`): The second word string.
+
         Returns:
             The scalar product of 2 words.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: specify the return type

File: paddlenlp/embeddings/token_embedding.py [hunk 6]
```diff
@@ -239,13 +250,27 @@ def search(self, words):
     def get_idx_from_word(self, word):
         """
         Get the index of specifying word by searching word_to_idx dict. 
+
+        Args:
+            word (`list` or `str` or `int`): The word which need to be get index.
+
+        Returns:
+            word_idx (`int`): The index of specifying word.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: word_idx (`int`) -> int

File: paddlenlp/embeddings/token_embedding.py [hunk 7]
```diff
@@ -218,19 +224,24 @@ def _extend_vocab(self, extended_vocab_path, vector_np, pad_vector,
...
         """
         Get the vectors of specifying words.
+
         Args:
-            words (object: `list` or `str` or `int`, required): The words which need to be searched.
+            words (`list` or `str` or `int`): The words which need to be searched.
+
         Returns:
-            word_vector (object: `numpy.array`): The vectors of specifying words.
+            word_vector (`numpy.array`): The vectors of specifying words.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: word_vector (`numpy.array`) -> numpy.array

File: paddlenlp/embeddings/token_embedding.py [hunk 8]
```diff
@@ -218,19 +224,24 @@ def _extend_vocab(self, extended_vocab_path, vector_np, pad_vector,
     def set_trainable(self, trainable):
         """
         Set the weight of embedding can be trained.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: Set the weight of embedding can be trained.
->
Whether or not to set the weights of token embedding to be trainable.

File: paddlenlp/embeddings/token_embedding.py (hunk 4 above)
Comment (yingyibiao) [reply paddlenlp/embeddings/token_embedding.py]: 可以简单介绍一下cosine similarity，比如取值范围之类的。

File: paddlenlp/embeddings/token_embedding.py (hunk 5 above)
Comment (yingyibiao) [reply paddlenlp/embeddings/token_embedding.py]: 可以简单介绍一下scalar product。

File: paddlenlp/embeddings/token_embedding.py [hunk 9]
```diff
@@ -239,13 +250,27 @@ def search(self, words):
...
     def get_idx_list_from_words(self, words):
         """
-        Get the index list of specifying words by searching word_to_idx dict. 
+        Get the index list of specifying words by searching word_to_idx dict.
+
+        Args:
+            words (`list` or `str` or `int`): The words which need to be get indexes.
+
+        Returns:
+            word_idxes (`list`): The indexes list of specifying words.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: word_idxes (`list`) -> list

File: paddlenlp/embeddings/token_embedding.py [hunk 10]
```diff
@@ -239,13 +250,27 @@ def search(self, words):
...
                                  self.unknown_token)
 
     def get_idx_list_from_words(self, words):
         """
-        Get the index list of specifying words by searching word_to_idx dict. 
+        Get the index list of specifying words by searching word_to_idx dict.
+
+        Args:
+            words (`list` or `str` or `int`): The words which need to be get indexes.
+
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: The words which need to be get indexes. 语法有问题
->
The input token words which we want to get the token indices converted from

File: paddlenlp/embeddings/token_embedding.py [hunk 11]
```diff
@@ -218,19 +224,24 @@ def _extend_vocab(self, extended_vocab_path, vector_np, pad_vector,
     def set_trainable(self, trainable):
         """
         Set the weight of embedding can be trained.
+
         Args:
-            trainable (object: `bool`, required):
+            trainable (`bool`):
                 Whether the weight of embedding can be trained.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: + The weights can be trained if `trainable` is set to True,  or the weights are fixed if `trainable` is False.

File: paddlenlp/transformers/bigbird/modeling.py [hunk 12]
```diff
@@ -260,6 +261,73 @@ def init_weights(self, layer):
...
+    This model is also a Paddle `paddle.nn.Layer <https://www.paddlepaddle.org.cn/documentation
+    /docs/en/api/paddle/fluid/dygraph/layers/Layer_en.html>`__ subclass. Use it as a regular Paddle Layer
+    and refer to the Paddle documentation for all matter related to general usage and behavior.
+
+    Args:
+        num_layers (`int`):
+            Number of hidden layers in the Transformer encoder.
+        vocab_size (`int`):
+            Vocabulary size of the XLNet model. Defines the number of different tokens that can
+            be represented by the `inputs_ids` passed when calling BigBirdModel.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: XLNet model -> BigBird model

File: paddlenlp/transformers/bigbird/modeling.py [hunk 13]
```diff
@@ -322,6 +390,67 @@ def forward(self,
...
+            attention_mask_list (`list`, optional):
+                A list which contains some tensors used in multi-head attention
+                to prevents attention to some unwanted positions, usually the
+                paddings or the subsequent positions. The tensors' shape will be
+                broadcasted to `[batch_size, n_head, sequence_length, sequence_length]`
+            rand_mask_idx_list (`list`, optional):
+                A list which contains some tensors used in bigbird random block.
+
+        Returns:
+            A tuple of shape (``encoder_output``, ``pooled_output``)
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: 需要介绍encoder_output和pooled_output

File: paddlenlp/transformers/bigbird/modeling.py [hunk 14]
```diff
@@ -425,6 +659,62 @@ def forward(self,
...
+            masked_positions (`List`):
+                The list of masked positions.
+
+        Returns:
+            A tuple of shape (``prediction_scores``, ``seq_relationship_score``).
+
+            - prediction_scores:
+                The logits of masked token prediction.
+            - seq_relationship_score:
+                The logits whether 2 sequences are NSP relationship.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: 需要说明一下prediction_scores和seq_relationship_score的shape和dtype

File: paddlenlp/transformers/bigbird/modeling.py [hunk 15]
```diff
@@ -402,12 +609,39 @@ def __init__(self,
...
+                The pooled output of BigBirdModel.
+            masked_positions (`Tensor`):
+                The list of masked positions.
+        Returns:
+            A tuple of shape (``prediction_scores``, ``seq_relationship_score``).
+
+            - prediction_scores:
+                The logits of masked token prediction.
+            - seq_relationship_score:
+                The logits whether 2 sequences are NSP relationship.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: 需要说明一下prediction_scores和seq_relationship_score的shape和dtype

File: paddlenlp/transformers/bigbird/modeling.py [hunk 16]
```diff
@@ -391,6 +582,22 @@ def forward(self, hidden_states, masked_positions=None):
...
+    Args:
+        hidden_size (`int`):
+            See :class:`BigBirdModel`.
+        vocab_size (`int`):
+            See :class:`BigBirdModel`.
+        activation (`str`):
+            See :class:`BigBirdModel`.
+        embedding_weights (`Tensor`, optional):
+            The weight of pretraining linear. Shape:[hidden_size, vocab_size].
+            If set to `None`, use random weight.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: pretraining linear -> pretrained embedding layer?

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 17]
```diff
@@ -41,7 +41,9 @@ class BigBirdTokenizer(PretrainedTokenizer):
         cls_token (str): The special token for cls. Default: "[CLS]".
         mask_token (str): The special token for mask. Default: "[MASK]".
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py:44]: Args前面加入空行

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 18]
```diff
@@ -156,6 +158,18 @@ def encode(self,
                max_pred_len=None,
                masked_lm_prob=0.15):
         """
+        Returns a tuple containing the encoded sequence and mask information.
+        Args:
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py:168]: Args前面插入空行

File: paddlenlp/transformers/bigbird/tokenizer.py (hunk 17 above)
Comment (yingyibiao) [reply paddlenlp/transformers/bigbird/tokenizer.py:44]: tokenizer的docstring格式有点问题，有些代码comment不了，可以在readthedocs预览页面看看展示效果哈

File: paddlenlp/embeddings/token_embedding.py (hunk 2 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 3 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: Done. 改为Specify unknown token

File: paddlenlp/embeddings/token_embedding.py (hunk 10 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: Done

File: paddlenlp/transformers/bigbird/tokenizer.py (hunk 18 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/tokenizer.py:168]: Done

Comment (joey12300) [reply paddlenlp/transformers/bigbird/tokenizer.py:164]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 4 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: Done

File: paddlenlp/transformers/bigbird/tokenizer.py (hunk 17 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/tokenizer.py:44]: done

File: paddlenlp/transformers/bigbird/modeling.py (hunk 16 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/modeling.py]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 11 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

File: paddlenlp/embeddings/token_embedding.py [hunk 19]
```diff
@@ -271,24 +296,33 @@ def _calc_word(self, word_a, word_b, calc_kernel):
...
+
         """
         dot = self._dot_np
         return self._calc_word(word_a, word_b, lambda x, y: dot(x, y))
 
     def cosine_sim(self, word_a, word_b):
         """
-        Calculate the cosine similarity of 2 words.
+        Cosine similarity is the cosine of the angle between two n-dimensional
+        vectors in an n-dimensional space. Calculate the cosine similarity of 2 word vectors.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: 两句话顺序换一下。

File: paddlenlp/embeddings/token_embedding.py [hunk 20]
```diff
@@ -271,24 +296,33 @@ def _calc_word(self, word_a, word_b, calc_kernel):
 
     def dot(self, word_a, word_b):
         """
-        Calculate the scalar product of 2 words.
+        Dot product or scalar product is an algebraic operation that takes
+        two equal-length sequences of numbers (usually coordinate vectors),
+        and returns a single number. Calculate the dot product of 2 words.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: 两句话顺序换一下。

File: paddlenlp/embeddings/token_embedding.py (hunk 19 above)
Comment (yingyibiao) [reply paddlenlp/embeddings/token_embedding.py]: 注意统一动词单复数，统一为第三人称单数。

File: paddlenlp/embeddings/token_embedding.py (hunk 20 above)
Comment (yingyibiao) [reply paddlenlp/embeddings/token_embedding.py]: 注意统一动词单复数，统一为第三人称单数。

File: paddlenlp/embeddings/token_embedding.py (hunk 10 above)
Comment (yingyibiao) [reply paddlenlp/embeddings/token_embedding.py]: The words which need to be get indexes. 语法有问题
->
The input token words which we want to get the token indices converted from

File: paddlenlp/embeddings/token_embedding.py [hunk 21]
```diff
@@ -239,13 +250,27 @@ def search(self, words):
...
+            `int`: The index of specifying word.
+
         """
         return get_idx_from_word(word, self.vocab.token_to_idx,
                                  self.unknown_token)
 
     def get_idx_list_from_words(self, words):
         """
-        Get the index list of specifying words by searching word_to_idx dict. 
+        Get the index list of specifying words by searching word_to_idx dict.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: 注意统一动词单复数，统一为第三人称单数。

File: paddlenlp/embeddings/token_embedding.py [hunk 22]
```diff
@@ -239,13 +250,27 @@ def search(self, words):
...
+
+        Args:
+            words (`list` or `str` or `int`): The words which need to be get indexes.
+
+        Returns:
+            `list`: The indexes list of specifying words.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py:272]: indexes -> indices

File: paddlenlp/embeddings/token_embedding.py [hunk 23]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
...
+        extended_vocab_path (`str`, optional):
             The file path of extended vocabulary.
-        trainable (object: `bool`, optional, default to True):
+            Default to `None`.
+        trainable (`bool`, optional):
             Whether the weight of embedding can be trained.
-        keep_extended_vocab_only (object: `bool`, optional, default to True):
+            Default to True.
+        keep_extended_vocab_only (`bool`, optional):
             Whether keep the extended vocabulary only, will be effective only if provides extended_vocab_path
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: Whether keep -> Whether to keep

File: paddlenlp/embeddings/token_embedding.py [hunk 24]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
...
+        embedding_name (`str`, optional):
             The pre-trained embedding model name. Use `paddlenlp.embeddings.list_embedding_name()` to
-            show which embedding model we have alreaady provide.
-        unknown_token (object: `str`, optional, default to `[UNK]`):
-            Specifying unknown token as unknown_token.
-        unknown_token_vector (object: list, optional, default to `None`):
+            list the names of all embedding models that we provide. 
+            Default to `w2v.baidu_encyclopedia.target.word-word.dim300`.
+        unknown_token (`str`, optional):
+            Specify unknown token.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: Specify unknown token. -> Specifies unknown token.
注意统一动词单复数，统一为第三人称单数。下同

**Review by yingyibiao — COMMENTED** [review]
注意统一动词格式，一般用第三人称单数。

File: paddlenlp/embeddings/token_embedding.py (hunk 19 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 20 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 22 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py:272]: 我看index的复数既可`indexes` 也可 `indices`，英美式差异

File: paddlenlp/embeddings/token_embedding.py [hunk 25]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
     by specifying extended_vocab_path.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py:47]: 上面的
def list_embedding_name():
    """
    List all names of pretrained embedding models paddlenlp provides.
    """

List -> Lists

File: paddlenlp/embeddings/token_embedding.py [hunk 26]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
...
     Args:
-        embedding_name (object: `str`, optional, default to `w2v.baidu_encyclopedia.target.word-word.dim300`):
+        embedding_name (`str`, optional):
             The pre-trained embedding model name. Use `paddlenlp.embeddings.list_embedding_name()` to
-            show which embedding model we have alreaady provide.
-        unknown_token (object: `str`, optional, default to `[UNK]`):
-            Specifying unknown token as unknown_token.
-        unknown_token_vector (object: list, optional, default to `None`):
+            list the names of all embedding models that we provide. 
+            Default to `w2v.baidu_encyclopedia.target.word-word.dim300`.
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: Default -> Defaults
下同

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 27]
```diff
@@ -29,19 +29,22 @@ class BigBirdTokenizer(PretrainedTokenizer):
     Constructs a BigBird tokenizer. It uses a basic tokenizer to do punctuation
     splitting, lower casing and so on, and follows a WordPiece tokenizer to
     tokenize as subwords.
+
     Args:
-        sentencepiece_model_file(str): file path of the vocabulary
-        do_lower_case (bool): Whether the text strips accents and convert to
+        sentencepiece_model_file (`str`): file path of the vocabulary
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py]: file -> File

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 28]
```diff
@@ -99,7 +102,7 @@ def vocab_size(self):
         """
         return the size of vocabulary.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py]: return -> Returns

Comment (yingyibiao) [reply paddlenlp/transformers/bigbird/tokenizer.py]: 后面加空行

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 29]
```diff
@@ -156,6 +159,19 @@ def encode(self,
...
                masked_lm_prob=0.15):
         """
+        Returns a tuple containing the encoded sequence and mask information.
+        
+        Args:
+            text (:obj:`str`, :obj:`List[str]` or :obj:`List[int]`):
+                The first sequence to be encoded. This can be a string, a list of strings (tokenized string using
+                the `tokenize` method) or a list of integers (tokenized string ids using the `convert_tokens_to_ids`
+                method)
+            max_seq_len (:obj:`int`, `optional`, defaults to :int:`None`):
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py]: defaults to :int:`None` -> defaults to `None`

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 30]
```diff
@@ -156,6 +159,19 @@ def encode(self,
...
+                If set to a number, will limit the total sequence returned so that it has a maximum length.
+                If set to None, will not limit the total sequence.
+            max_pred_len (:obj:`int`, `optional`, defaults to :int:`None`):
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py]: defaults to :int:`None` -> defaults to `None`

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 31]
```diff
@@ -259,7 +275,7 @@ def num_special_tokens_to_add(self, pair=False):
                 number of added tokens in the case of a single sequence if set to False.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py]: specify pair data type

File: paddlenlp/transformers/bigbird/tokenizer.py [hunk 32]
```diff
@@ -284,7 +300,7 @@ def build_inputs_with_special_tokens(self, token_ids_0, token_ids_1=None):
                 Optional second list of IDs for sequence pairs.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/tokenizer.py:304]: 上面的显示有点问题：
            - single sequence: ``[CLS] X [SEP]``
            - pair of sequences: ``[CLS] A [SEP] B [SEP]``

File: paddlenlp/transformers/bigbird/modeling.py [hunk 33]
```diff
@@ -260,6 +261,73 @@ def init_weights(self, layer):
 
 @register_base_model
 class BigBirdModel(BigBirdPretrainedModel):
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py:263]: Default -> Defaults

Comment (yingyibiao) [reply paddlenlp/transformers/bigbird/modeling.py:263]: 下同

File: paddlenlp/transformers/bigbird/modeling.py [hunk 34]
```diff
@@ -448,6 +773,86 @@ def __init__(self, vocab_size, use_nsp=False, ignore_index=0):
     def forward(self, prediction_scores, seq_relationship_score,
                 masked_lm_labels, next_sentence_labels, masked_lm_scale,
                 masked_lm_weights):
+        r"""
+        The BigBirdPretrainingCriterion forward method, overrides the __call__() special method.
+
+        Args:
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py:782]: specify the shape and dtype of tensors

File: paddlenlp/transformers/bigbird/tokenizer.py (hunk 32 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/tokenizer.py:304]: image
已修复

File: paddlenlp/transformers/bigbird/modeling.py [hunk 35]
```diff
@@ -425,6 +677,66 @@ def forward(self,
                 position_ids=None,
                 rand_mask_idx_list=None,
                 masked_positions=None):
+        r"""
+        The BigBirdForSequenceClassification forward method, overrides the __call__() special method.
+
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: BigBirdForSequenceClassification -> BigBirdForPretraining

File: paddlenlp/embeddings/token_embedding.py [hunk 36]
```diff
@@ -217,35 +223,54 @@ def _extend_vocab(self, extended_vocab_path, vector_np, pad_vector,
...
         return self(idx_tensor).numpy()
 
     def get_idx_from_word(self, word):
         """
-        Get the index of specifying word by searching word_to_idx dict. 
+        Gets the index of specifying word by searching word_to_idx dict. 
+
+        Args:
+            word (`list` or `str` or `int`): The input token words which we want to get the token indices converted from.
+
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: words -> word
indices -> index

File: paddlenlp/transformers/bigbird/modeling.py [hunk 37]
```diff
@@ -260,6 +261,73 @@ def init_weights(self, layer):
...
+            Defaults to ``0.1``.
+        dim_feedforward (`int`, optional):
+            Dimensionality of the "intermediate" (often named feed-forward) layer in the Transformer encoder.
+            Defaults to ``3072``.
+        activation (`str`, optional):
+            The non-linear activation function in the feed-forward layer.
+            ``"gelu"``, ``"relu"``, ``"silu"`` and ``"gelu_new"`` are supported.
+            Defaults to ``"gelu"``.
+        normalize_before (`bool`, optional):
+            Indicate whether to put layer normalization into preprocessing of MHA and FFN sub-layers.
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: Indicate -> Indicates

File: paddlenlp/transformers/bigbird/modeling.py [hunk 38]
```diff
@@ -260,6 +261,73 @@ def init_weights(self, layer):
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py:261]: 上面的""" Initialization hook """可以用#注释

Comment (joey12300) [reply paddlenlp/transformers/bigbird/modeling.py:261]: done

File: paddlenlp/transformers/bigbird/modeling.py (hunk 37 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/modeling.py]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 36 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

File: paddlenlp/transformers/bigbird/modeling.py (hunk 35 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/modeling.py]: done

File: paddlenlp/transformers/bigbird/modeling.py [hunk 39]
```diff
@@ -437,6 +749,22 @@ def forward(self,
...
+            See :class:`BigBirdModel`.
+        use_nsp (`bool`, optional):
+            It decides whether it considers NSP loss.
+            Defaults: False
+        ignore_index (`int`):
+            Specifies a target value that is ignored and does
+            not contribute to the input gradient. Only valid
+            if :attr:`soft_label` is set to :attr:`False`.
+            Defaults: kIgnoreIndex(0)
+    """
```
Comment (yingyibiao) [inline paddlenlp/transformers/bigbird/modeling.py]: Defaults: 0.

File: paddlenlp/embeddings/token_embedding.py [hunk 40]
```diff
@@ -46,20 +46,26 @@ class TokenEmbedding(nn.Embedding):
...
             The file path of extended vocabulary.
-        trainable (object: `bool`, optional, default to True):
+            Defaults to `None`.
+        trainable (`bool`, optional):
             Whether the weight of embedding can be trained.
-        keep_extended_vocab_only (object: `bool`, optional, default to True):
-            Whether keep the extended vocabulary only, will be effective only if provides extended_vocab_path
+            Defaults to True.
+        keep_extended_vocab_only (`bool`, optional):
+            Whether to keep the extended vocabulary only, will be effective only if provides extended_vocab_path
```
Comment (yingyibiao) [inline paddlenlp/embeddings/token_embedding.py]: 行末尾加一个句号

File: paddlenlp/transformers/bigbird/modeling.py (hunk 39 above)
Comment (joey12300) [reply paddlenlp/transformers/bigbird/modeling.py]: done

File: paddlenlp/embeddings/token_embedding.py (hunk 40 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

**Review by yingyibiao — APPROVED** [review]
LGTM

**Pull request merged**